dvc/external_repo.py

```diff
@@ -69,6 +69,18 @@ def _external_repo(url=None, rev=None, cache_dir=None):
 
     repo = Repo(new_path)
     try:
+        # check if the URL is local and no default remote is present
+        # add default remote pointing to the original repo's cache location
+        if os.path.isdir(url):
+            rconfig = RemoteConfig(repo.config)
+            if not _remote_config_exists(rconfig):
+                rconfig.add(
+                    "upstream",
+                    repo.cache.local.cache_dir,
```

Comment:
You should point to the _original repo_ cache location, not to the cache of the cloned repo, since it is empty.